apps/src/codebridge/Workspace/WorkspaceAndOutput.tsx

```diff
@@ -38,44 +40,79 @@ const WorkspaceAndOutput: React.FunctionComponent = () => {
   }, []);
 
   useEffect(() => {
-    normalizeConsoleHeight(consoleHeight);
-  }, [consoleHeight, columnHeight]);
+    normalizeOutputHeight(outputHeight, miniApp);
+  }, [outputHeight, columnHeight, miniApp]);
 
   const handleResize = (desiredHeight: number) => {
     // While the horizontal resizer thinks it's resizing the content above it, which
-    // is the editor panel, we are actually storing the size of the console below it.
-    // That way, if the window resizes, the console stays the same height while the editor
+    // is the editor panel, we are actually storing the size of the output below it.
+    // That way, if the window resizes, the output stays the same height while the editor
     // changes in height.
-    const consoleDesiredHeight = columnHeight - desiredHeight;
-    normalizeConsoleHeight(consoleDesiredHeight);
+    const desiredOutputHeight = columnHeight - desiredHeight;
+    normalizeOutputHeight(desiredOutputHeight, miniApp);
   };
 
-  // Given a desired console height, ensure it is between the minimum and maximum
-  // console height.
-  const normalizeConsoleHeight = (desiredConsoleHeight: number) => {
+  // Given a desired output height, ensure it is between the minimum and maximum
+  // output height.
+  const normalizeOutputHeight = (
+    desiredOutputHeight: number,
+    miniAppName: string | undefined
+  ) => {
     // Minimum height fits 4 lines of text.
-    const consoleHeightMin = 120;
-    const consoleHeightMax = window.innerHeight - 200;
-
-    setConsoleHeight(
-      Math.max(
-        consoleHeightMin,
-        Math.min(desiredConsoleHeight, consoleHeightMax)
-      )
+    const outputHeightMin = 120;
+    const outputHeightMax = window.innerHeight - 200;
```

Comment:
What does `200` represent?

Reply:
It's the minimum size of the editor. I can add a constant for that.